code/widgets/org.eclipse.scout.jswidgets.ui.html/src/main/resources/org/eclipse/scout/jswidgets/ui/html/texts/Texts.properties

```diff
@@ -101,6 +101,7 @@ RevealTooltip=Brings the widget into view by scrolling the first scrollable pare
 Salutation=Salutation
 SaveMenuTooltip=Saves the changes without closing the form.
 Settings=Settings
+SliderField=Slider Field
```

Comment:
It would not add a text key for this (there is only one language anyway). It is not even used in WidgetsOutlineModel.ts.